packages/design-system-react/tsconfig.build.json

```diff
@@ -4,7 +4,13 @@
     "baseUrl": "./",
     "outDir": "./dist",
     "rootDir": "./src",
-    "jsx": "react"
+    "jsx": "react",
+    "types": [
+      "react",
+      "jest",
+      "@testing-library/react",
+      "@testing-library/jest-dom"
+    ]
```

Comment:
- `@testing-library/react`: Provides TypeScript type definitions for React Testing Library. This allows TypeScript to understand the types used in our tests, enabling features like autocomplete and type checking.
- `@testing-library/jest-dom` : Offers custom Jest matchers that are tailored for DOM node assertions. Including its types ensures that these matchers are recognized and type-checked by TypeScript.

Without we get this build error running `yarn build`

<img width="675" alt="Screenshot 2024-10-31 at 3 07 47 PM" src="https://github.com/user-attachments/assets/19efc5aa-6bc9-4af0-a541-00743a9915ed">